packages/shared/components/AccountSwitcher.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="typescript">
+    import type { WalletAccount } from 'lib/typings/wallet'
+    import { AccountSwitcherModal, Icon, Text } from 'shared/components'
+    import { activeProfile, getColor } from 'shared/lib/profile'
+    import { selectedAccount } from 'shared/lib/wallet'
+
+    export let accounts: WalletAccount[] = []
+    export let onCreateAccount = (..._: any[]): void => {}
+
+    let showModal: boolean = false
+
+    function toggleModal() {
+        showModal = !showModal
+    }
+</script>
+
+<button on:click={toggleModal} class="flex flex-row justify-center items-center space-x-2">
```

Comment:
When I click the chevron to close the modal, it closes and reopens

Reply:
I know 😭 This happens with all the modals, I really tried yesterday refactoring the modal to blacklist some dom elements from doing this but modals use a custom action so they close when you click outside them and then this happens. If you click the network status or your profile at the bottom left we have the same situation, maybe we should create a task to improve this

Reply:
https://github.com/iotaledger/firefly/issues/2434